client/domain/ReinforcementObjectFactory.js

```diff
@@ -21,15 +21,15 @@ tie.factory('ReinforcementObjectFactory', [
   'ReinforcementBulletObjectFactory', function(ReinforcementBulletObjectFactory) {
     var Reinforcement = function(task) {
       this._task = task;
-      this._passedList = {};
-      this._pastFailsList = {};
+      this._passedTagsList = {};
```

Comment:
One other thing I forgot to mention earlier (sorry) -- can we call these this._passedTags and this._pastFailedCases? They're not lists; lists are ordered.